feat(email-templates): refactor and add email templates from 1.48.3 (#2758)
FusionAuth/fusionauth-site: astro/src/content/docs/customize/email-and-messages/templates-replacement-variables.mdx

import APIBlock from 'src/components/api/APIBlock.astro';
import APIField from 'src/components/api/APIField.astro';
import EnterpriseEditionBlurb from 'src/content/docs/_shared/_enterprise-edition-blurb.astro';
import BreachedPasswordHTML from 'src/content/docs/_shared/email/_breached-password-notification-html.mdx';
import BreachedPasswordText from 'src/content/docs/_shared/email/_breached-password-notification-txt.mdx';
import PasswordlessHTML from 'src/content/docs/_shared/email/_passwordless-login-html.mdx';

> import PasswordlessText from 'src/content/docs/_shared/email/_passwordless-login-txt.mdx';

Reviewer: Please update this page to use the remote include components for all the other sample messages.

import COPPANoticeReminderText from 'src/content/docs/_shared/email/_coppa-notice-reminder-txt.mdx';
import ForgotPasswordHTML from 'src/content/docs/_shared/email/_forgot-password-html.mdx';
import ForgotPasswordText from 'src/content/docs/_shared/email/_forgot-password-txt.mdx';
import TwoFactorAuthenticationHTML from 'src/content/docs/_shared/email/_two-factor-authentication-html.mdx';
import TwoFactorAuthenticationText from 'src/content/docs/_shared/email/_two-factor-authentication-txt.mdx';
import EmailVerificationHTML from 'src/content/docs/_shared/email/_email-verification-html.mdx';
import EmailVerificationText from 'src/content/docs/_shared/email/_email-verification-txt.mdx';
import ThreatDetectedHTML from 'src/content/docs/_shared/email/_threat-detected-html.mdx';
import ThreatDetectedText from 'src/content/docs/_shared/email/_threat-detected-txt.mdx';
import TwoFactorAuthenticationMethodAddedHTML from 'src/content/docs/_shared/email/_two-factor-authentication-method-added-html.mdx';
import TwoFactorAuthenticationMethodAddedText from 'src/content/docs/_shared/email/_two-factor-authentication-method-added-txt.mdx';

> 

Reviewer: ```suggestion

[//]: # "MAKE SURE YOU UPDATE astro/src/content/docs/_shared/email/template_url_list if you add any templates"

```

Instead, the attempt will fail and there will be an error added to the Event Log.
</Aside>

> 

Reviewer: ```suggestion

### Retrieving Default Templates
```

> To get all the latest email templates at once, you can run the following command in your desired destination directory:

Reviewer: ```suggestion

In order to version control the templates or customize them, you can use the admin UI. But you can also pull retrieve them all by using the following command:
```

```
wget -i https://raw.githubusercontent.com/FusionAuth/fusionauth-site/master/astro/src/content/docs/_shared/email/template_url_list

> ```

Reviewer: ```suggestion

This will place all the email templates in the current working directory.
```
